apps/web/lib/event-types/[type]/getServerSideProps.ts

```diff
@@ -19,48 +23,29 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const typeParam = parseInt(asStringOrThrow(query.type));
 
   if (Number.isNaN(typeParam)) {
-    const notFound = {
-      notFound: true,
-    } as const;
-
-    return notFound;
+    notFound();
```

Comment:
My mistake, it should be returned.

```suggestion
    return notFound();
```